pkg/pidfile/pidfile.go

```diff
@@ -36,10 +36,10 @@ func New(path string) (*PIDFile, error) {
 		return nil, err
 	}
 	// Note MkdirAll returns nil if a directory already exists
-	if err := system.MkdirAll(filepath.Dir(path), os.FileMode(0755)); err != nil {
+	if err := system.MkdirAll(filepath.Dir(path), os.FileMode(0o755)); err != nil {
```

Comment:
Looks like casting to `os.FileMode` is redundant here as well; let me update this commit to remove it.

Reply:
[]()